tvloader/parser2v1/parse_file_test.go

```diff
@@ -885,4 +886,60 @@ func TestParser2_1FailsIfArtifactURIBeforeArtifactName(t *testing.T) {
 	}
 }
 
+func TestParser2_1FilesWithoutSpdxIdThrowError(t *testing.T) {
+	//case 1
```

Comment:
[optional] it is often idiomatic to put a space after double slashes for comments.